gemini-resin-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -1,31 +1,42 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+  <modelVersion>4.0.0</modelVersion>
 
-    <groupId>${groupId}</groupId>
-    <artifactId>${artifactId}</artifactId>
-    <version>${version}</version>
-    <packaging>jar</packaging>
+  <groupId>${groupId}</groupId>
+  <artifactId>${artifactId}</artifactId>
+  <version>${version}</version>
+  <packaging>jar</packaging>
 
-    <name>A custom Gemini project</name>
+  <name>A custom Gemini project</name>
 
-    <properties>
-        <maven.compiler.source>10</maven.compiler.source>
-        <maven.compiler.target>10</maven.compiler.target>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    </properties>
+  <properties>
+    <maven.compiler.source>10</maven.compiler.source>
+    <maven.compiler.target>10</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+  </properties>
 
-    <dependencies>
-        <dependency>
-            <groupId>com.techempower</groupId>
-            <artifactId>gemini</artifactId>
-            <version>${geminiVersion}</version>
-        </dependency>
-        <dependency>
-            <groupId>com.techempower</groupId>
-            <artifactId>gemini-resin</artifactId>
-            <version>${geminiVersion}</version>
-        </dependency>
-    </dependencies>
+  <dependencies>
+    <dependency>
+      <groupId>com.techempower</groupId>
+      <artifactId>gemini</artifactId>
+      <version>${geminiVersion}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.techempower</groupId>
+      <artifactId>gemini-resin</artifactId>
+      <version>${geminiVersion}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.techempower</groupId>
+      <artifactId>gemini-jdbc</artifactId>
+      <version>${geminiVersion}</version>
+    </dependency>
+    <!-- gemini-hikaricp is an alternative to gemini-jdbc -->
+    <!-- <dependency> -->
+    <!-- <groupId>com.techempower</groupId> -->
+    <!-- <artifactId>gemini-hikaricp</artifactId> -->
+    <!-- </dependency> -->
```

Comment:
I think this could be the other way around - if my instinct is "my new Gemini projects will always use hikaricp," then I think we should support that recommendation by default.

Reply:
I don't disagree, however the HikariCP integration hasn't received any real-world usage yet outside of me doing TFB tests. Happy to switch it, but I didn't want to presume.